pkg/timeutils/utils.go

```diff
@@ -9,12 +9,8 @@ import (
 // or Unix timestamp, if successful returns a Unix timestamp
 // as string otherwise returns value back.
 func GetTimestamp(value string) string {
-	format := RFC3339NanoFixed
 	loc := time.FixedZone(time.Now().Zone())
-	if len(value) < len(format) {
```

Comment:
If you're going to remove that check then I think we need to add new testcases to make sure that short timestamps are still going to work.